pkg/config/configobserver/configobserver.go

```diff
@@ -139,6 +139,36 @@ func (c *Controller) retrieveConfig(ctx context.Context) error {
 		if noproxy, ok := secret.Data["noProxy"]; ok {
 			nextConfig.HTTPConfig.NoProxy = string(noproxy)
 		}
+		if reportEndpoint, ok := secret.Data["reportEndpoint"]; ok {
+			nextConfig.ReportEndpoint = string(reportEndpoint)
+		}
+		if reportPullingDelay, ok := secret.Data["reportPullingDelay"]; ok {
+			nextConfig.ReportPullingDelay, err = time.ParseDuration(string(reportPullingDelay))
+			if err != nil {
+				klog.Warningf(
+					"reportPullingDelay secret contains an invalid value (%s). Using previous value",
+					reportPullingDelay,
+				)
+			}
+		}
+		if reportPullingTimeout, ok := secret.Data["reportPullingTimeout"]; ok {
+			nextConfig.ReportPullingTimeout, err = time.ParseDuration(string(reportPullingTimeout))
```

Comment:
Ufortunatelly you would have to use it with if. time.ParseDuration when fails is not returning the original value, but 0. See https://play.golang.com/p/-WRrTb5Xg5s
Correct would be:
```
if v, err := time.ParseDuration(string(reportPullingTimeout); err == nil {
  nextConfig.ReportPullingTimeout = v
}
```

Reply:
It looks this is still not implemented no ?

Reply:
I implemented in a different way, because I didn't get the point of your comment. Nxt commit will fix that. TYVM